packages/flutter/test/material/text_field_test.dart

```diff
@@ -854,6 +854,53 @@ void main() {
     expect(find.text('CUT'), findsNothing);
   });
 
+  testWidgets('text field build empty tool bar when no options available ios', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData(platform: TargetPlatform.iOS),
+          home: const Material(
+            child: TextField(
+              readOnly: true,
+            ),
+          ),
+        )
+    );
+
+    await tester.tap(find.byType(TextField));
+    await tester.pump(const Duration(milliseconds: 50));
+
+    await tester.tap(find.byType(TextField));
+    await tester.pump(const Duration(milliseconds: 50));
```

Comment:
Could you make this a `pumpAndSettle`?  Since I think you're waiting for the menu to be able to animate in here.